src/elements/content-preview/ContentPreview.scss

```diff
@@ -60,41 +60,26 @@
         }
     }
 
-    &.be-is-small {
-        .bcpr-body {
-            position: relative;
-        }
-
-        .bcs.bcs-is-open {
-            position: absolute;
-            right: 0;
-        }
```

Comment:
I removed this entire block as it's no longer needed. the `breakpoint($medium-screen)` blocks will handle the styles for all cases of `be-is-small`.

for example, `.bcpr-body { position: relative; }` is overwritten in the block below. the `.bcs.bcs-is-open` styles are handled in ContentSidebar.scss